apps/omg_watcher/lib/utxo_selection.ex

```diff
@@ -17,7 +17,7 @@ defmodule OMG.Watcher.UtxoSelection do
   Provides Utxos selection and merging algorithms.
   """
 
-  alias OMG.API.{Crypto, State.Transaction}
+  alias OMG.{Crypto, State.Transaction}
```

Comment:
hm... we normally don't write aliases this way

Reply:
yeah, I'm not a fan either, but I was just moving what I found, I don't want to add that to the bulk of changes here.

To enforce a certain style, we'd probably need a `credo` check